src/client/pages/store-management/store-management-view.html

```diff
@@ -0,0 +1,32 @@
+<div class="store-management-view">
+    <div class="container is-fullhd">
+        <section>
+            <div class="hero is-primary">
+                <div class="title is-1">Store Management</div>
+            </div>
+            <div class="columns is-grouped">
```

Comment:
May I know why you use is-grouped

Reply:
I was trying out something, then I forgot to change.